### ISSUE TITLE
Internal note is displayed on the invoice

### ISSUE THREAD

**Issue by cfrancoae** [odoo/odoo#103716]
Impacted versions: v13.0

Steps to reproduce:
1. Edit an invoice
2. Add text to Narration field
3. Save invoice
4. Print invoice

Current behavior:
We have seen that the "Narration" field with the "Add an internal note" placeholder is displayed when printing the invoice for customers and suppliers.

Expected behavior:
We understand that since we are talking about an internal note, it shouldn't be displayed on the invoice.

Video/Screenshot link (optional):

Support ticket number submitted via odoo.com/help (optional):

**Comment by niyasraphy** [comment]
@cfrancoae as per Odoo 16,  at the invoice level the place holder is "Terms and Conditions" right ? so seems fine to be printed along with the invoice.

**Comment by cfrancoae** [comment]
Yeees! You are right, but in the view, this field has a placeholder "Add an internal note..." like the next picture.
![image](https://user-images.githubusercontent.com/82881462/198208709-0956ab0f-292a-4dfa-876d-2991e518d097.png)

And I think, this is the mistake.

**Comment by niyasraphy** [comment]
@cfrancoae this is in journal entries menu right ?
@lordkrandel , can i create a PR to have same placeholder for the field in invoice and JE view ?

**Comment by cfrancoae** [comment]
@niyasraphy there is in the customer and supplier view, when you create/edit an invoice.
![image](https://user-images.githubusercontent.com/82881462/198211182-0e651156-4fba-448d-9e3c-9553421d7220.png)

Then, at the model, this fields is "Terms and Conditions" but in the view has the placeholder "Add an internal note...", but this "internal note" is printed on the invoice (not is an "internal note").
I think that is an error.

**Comment by niyasraphy** [comment]
@cfrancoae in the invoice the place holder is "Terms and Conditions" , may be in your case translation causing the issue ?
https://github.com/odoo/odoo/blob/16.0/addons/account/views/account_move_views.xml#:~:text=%3Cfield%20name%3D%22narration%22%20placeholder%3D%22Terms%20and%20Conditions%22%20colspan%3D%222%22%20nolabel%3D%221%22/%3E

**Comment by cfrancoae** [comment]
We are working with Odoo 13. In our account_move.py model, has the next field.

![image](https://user-images.githubusercontent.com/82881462/198225360-45771c76-d835-4cce-87d0-dbf0d2916c1b.png)

But in account_move_views.xml (Odoo Core), this fields has the placeholder, I said before. And we have 2 times this fields with this placeholder.

![image](https://user-images.githubusercontent.com/82881462/198226500-612886e0-e294-415c-b2b6-8b5e59ae8acf.png)
![image](https://user-images.githubusercontent.com/82881462/198226575-0ca2cb65-41a9-4d87-a12e-670b5ec7be14.png)

I saw the translation file and this is what we have:

![image](https://user-images.githubusercontent.com/82881462/198226866-abc2d58c-9cf0-465c-b25c-de0321c9d3ce.png)

**Comment by niyasraphy** [comment]
@cfrancoae so the issue is in 13.0, i think it will be better to update the version in issue(message)

**Comment by cfrancoae** [comment]
What you say is correct.
I looked at it wrong, the "narration" field is used twice in the view.
In version 13, both times the field is used, the placeholder is set to "Add an internal note...", as of version 14, one of the two placeholders was changed to "Terms and conditions".

In this way and seeing it modified in versions later than the one we are using, we understand that it is a bug already detected and modified from version 14 of Odoo.

For our part, we will modify this placeholder so that it does not appear as "Internal note".
